mapie/regression.py

```diff
@@ -658,7 +661,7 @@ def predict(
                         )
                         for _alpha in alpha_
                     ]
-                )
+                ).data
```

Comment:
I didn't understand the use of this object.

Reply:
Without `.data`, the output is a masked array (with as mask full of True). This restore pristine numpy arrays.

Reply:
Okay, good to know !